google_redis/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 3.0"
```

Comment:
Could you change this to 4.0?

Reply:
Also, could you add the `google-beta` provider?

Reply:
added!